recipes/monovar/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "MonoVar" %}
+# MonoVar doesn't have versions, so I'm just freezing the commit 7b47571d85a377c425feea86f6e591ee832fe342 as 'bioconda'
+{% set version = "bioconda" %}
```

Comment:
Sure. My thought behind this was, that this is not any real version and in case the creator wants to start versioning, I don't want to mess that up. But then I guess that 0.0.1 should always be smaller!